src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.johan.blignaut.entelect.vitalityincubator.task1;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ApplicationConfiguration {
```

Comment:
incorrect annotation here, this is your configuration class, this is where you should be defining your beans and dependencies

please check the difference between @bean, @Configration and @Componant, where to use.